plugins/logger.py

```diff
@@ -12,7 +12,8 @@ class DiscordLogger(BasePlugin):
                 "message_delete",
                 "message_edit",
                 "member_join",
-                "member_remove"
+                "member_remove",
+                "log_event"
```

Comment:
I would prefer that you add a separate event name for each event, for better control.